Formula/jmeter.rb

```diff
@@ -1,7 +1,7 @@
 class Jmeter < Formula
   desc "Load testing and performance measurement application"
   homepage "https://jmeter.apache.org/"
-  url "https://www.apache.org/dist/jmeter/binaries/apache-jmeter-5.1.1.tgz"
+  url "https://www.apache.org/dyn/closer.cgi?path=jmeter/binaries/apache-jmeter-5.1.1.tgz"
```

Comment:
@fxcoudert This needs to be reverted. The link you provided does not download a tarball.